src/lib/Preview.js

```diff
@@ -1943,6 +2006,19 @@ class Preview extends EventEmitter {
             tokenOrTokenMap => tokenOrTokenMap[this.file.id],
         );
     };
+
+    /**
+     * Get a thumbnail image element
+     *
+     * @param {number} pageNumber - the page number
+     * @return {Promise} - promise resolves with the image HTMLElement or null if generation is in progress
+     */
+    getThumbnail(pageNumber) {
```

Comment:
Is this method currently used anywhere?

Reply:
It would be called by BUIE to get a Thumb representation from a page